xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-test/xwiki-platform-mentions-test-docker/src/test/it/org/xwiki/mentions/test/ui/MentionsIT.java

```diff
@@ -67,6 +74,20 @@
         void run() throws Exception;
     }
 
+    @AfterEach
+    void tearDown(TestUtils setup, TestReference reference) throws Exception
+    {
+        // clear all notifications between each test.
+        runAsUser(setup, U2_USERNAME, USERS_PWD, () -> {
```

Comment:
Why not deleting the users between test instead?

Reply:
Just clearing the notifications is a lighter operation, but overall the result would be the same.

Reply:
Actually tearDown is bad practice in general. You should have a setup instead. And a single one since it's a scenario.